packages/firebase_storage/lib/firebase_storage.dart

```diff
@@ -48,6 +48,13 @@ class StorageReference {
     );
   }
 
+  Future<dynamic> getDownloadURL() async {
```

Comment:
you could make this not be `async` and remove the `await`

Reply:
I was mistaken, this is actually necessary in strong mode

Reply:
is the `dynamic` really needed ?